packages/commitlint-plugin-jira/src/rules/jira-project-key.ts

```diff
@@ -0,0 +1,32 @@
+import { Rule } from '@commitlint/types';
+
+export const JIRA_PROJECT_KEY = 'jira-project-key';
+
+export const jiraProjectKey: Rule<string[]> = ({ subject }, applicable, projects) => {
+  const invert = applicable === 'never';
+  const jiraTicket = subject?.match(/[A-Z]{2,}-\d+/);
+
+  if (projects && projects.length) {
+    if (jiraTicket && jiraTicket.length === 1) {
+      const projectsStr = projects.join(', ');
+      const containsKey = projects.reduce(
+        (acc, project) => acc || jiraTicket[0].includes(project),
+        false,
+      );
```

Comment:
Why are we using `reduce` for this instead of `some`?

Reply:
Removed this rule.